docs/Install.md

```diff
@@ -130,6 +130,8 @@ Configure and compile DPDK
 
     The [install script](../scripts/install.sh) will automatically run the [environment setup script](../scripts/setup_environment.sh), which configures your local environment.  This should be run once for every reboot, as it loads the appropraite kernel modules and can bind your NIC ports to the DPDK driver.
 
+    If `which python` shows **not found** in your environment, please link `python` it to your Python2 or Python3, e.g., via `link -s` or `alias python=$(which python3)`.
```

Comment:
```suggestion
    If `which python` shows **not found** in your environment, please link `python` to Python3, e.g., via `link -s` or `alias python=$(which python3)`.
```

Reply:
I think maybe changing the wording to this might let people know that Python2 isn't really relevant anymore.

Reply:
Looks great! I've modified it 😁